planning/autoware_obstacle_cruise_planner/src/node.cpp

```diff
@@ -495,13 +549,15 @@ void ObstacleCruisePlannerNode::onTrajectory(const Trajectory::ConstSharedPtr ms
 {
   const auto ego_odom_ptr = ego_odom_sub_.takeData();
   const auto objects_ptr = objects_sub_.takeData();
+  const auto pointcloud_ptr = pointcloud_sub_.takeData();
```

Comment:
Could you change to call this function only if necessary? Computing cost of this function is not small enough